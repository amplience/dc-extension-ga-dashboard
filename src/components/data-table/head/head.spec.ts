import { render } from '@testing-library/svelte';
import Head from './head.svelte';

describe('Head', () => {
  it('should render the DataTable Head component', () => {
    const { container } = render(Head, {});

    expect(container.firstChild).toMatchSnapshot();
  });
});
