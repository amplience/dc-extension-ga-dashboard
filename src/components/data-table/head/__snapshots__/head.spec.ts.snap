// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Head should render the DataTable Head component 1`] = `
<div>
  <div
    class="data-table-head svelte-1nivie3"
  />
</div>
`;
